addons/tacphone/apps/bft/functions/fnc_createApp.sqf

```diff
@@ -0,0 +1,46 @@
+#include "..\script_component.hpp"
+/*
+ * Author: KJW
+ * Initializes the App
+ *
+ * Arguments:
+ * TacPhone main display
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_tacphone_bft_fnc_createApp
+ *
+ * Public: No
+ */
+
+params ["_display", "_appSection"];
+
+private _fullSize = [0, 0, (ctrlPosition _appSection)#2, (ctrlPosition _appSection)#3];
+
+GVAR(appsection) = _display ctrlCreate ["RscControlsGroupNoScrollbars", -1, _appSection];
+GVAR(appsection) ctrlSetPosition _fullSize;
+GVAR(appsection) ctrlCommit 0;
+
+_map = _display ctrlCreate [QEGVAR(tacphone,mapControl), -1, GVAR(appsection)];
```

Comment:
cannot put map controls into control groups
e.g. map is not correctly placed at 1280x1024/small res